inc/Ui/Editor.php

```diff
@@ -63,7 +63,7 @@ public function show()
         }
 
         // create the Editor form
-        $form = new Form(['action' => wl($ID), 'id' => 'dw__editform']);
+        $form = new Form(['id' => 'dw__editform']);
```

Comment:
I’m not sure, but my impression was that the action attribute should not be left out. If I see it right it is even added at some other places in the code last years.

Reply:
Ah, I see the form has already a default action.